Financely/basic_app/stock_data.py

```diff
@@ -11,6 +11,7 @@ def candlestick_data(ticker):
     # newdata= data.to_dict()
     # print(newdata)'%Y-%m-%d'
     old  =  aapl.history(start="2001-05-21", end=now)
+    new = aapl.hisory(start="200020")
```

Comment:
**Issue**: Typo in method name

**Reason**: Method 'hisory' does not exist, should be 'history'.

**Suggestion**: Change 'aapl.hisory(start="200020")' to 'aapl.history(start="2000-01-01")'

Reply:
ok thank you

Reply:
You're welcome! Just to confirm, the typo in the method name has been noted. The method `hisory` should be corrected to `history`. 

Here's what needs to be fixed:
- Change `'aapl.hisory(start="200020")'` to `'aapl.history(start="2000-01-01")'`

This change is necessary because the method `hisory` does not exist in the library you're using. The correct method name is `history`, which retrieves historical stock data.

If you have any more questions or need further assistance, feel free to ask!

Reply:
ok i will change it

Reply:
✅ Thread resolved